### ISSUE TITLE
Signing app for Gatekeeper

### ISSUE THREAD

**Issue by Diddern** [toy/blueutil#68]
Currently users must disable Gatekeeper to use the util. 
Is it possible to sign the app according to Apple guidelines so that it can be used without disabling Gatekeeper?
https://developer.apple.com/developer-id/

**Comment by Diddern** [comment]
I see now that this has been discussed in https://github.com/toy/blueutil/issues/23

**Comment by toy** [comment]
About disabling Gaetkeeper, you mean when using version compiled from source, version from Homebrew or MacPorts?

**Comment by Diddern** [comment]
Yes. When using version from Homebrew it should not require disableing of security features.
